modules/overviews/app/components/project_custom_fields/sections/edit_component.html.erb

```diff
@@ -0,0 +1,41 @@
+<%=
+  component_wrapper do
+    primer_form_with(
+      model: @project,
+      method: :put,
+      data: { turbo: true, turbo_stream: true, "test-selector": "async-dialog-content" },
+      url: update_project_custom_values_path(project_id: @project.id, section_id: @project_custom_field_section.id),
+    ) do |f|
+      component_collection do |collection|
+        collection.with_component(Primer::Alpha::Dialog::Body.new(my: 3, classes: "Overlay-body_autocomplete_height")) do
+          render(Projects::CustomFields::Form.new(
+            f,
+            project: @project,
+            custom_field_section: @project_custom_field_section,
+            wrapper_id: "#edit-project-custom-fields-dialog-#{@project_custom_field_section.id}"
+          ))
+        end
+        collection.with_component(Primer::Alpha::Dialog::Footer.new) do
```

Comment:
Same as above. If we can use the footer correctly, we'll also benefit from the correct scrolling behavior on longer modals. Currently, we still have to scroll the whole content to see the "Save" and "Cancel" buttons.